service/namespace.go

```diff
@@ -11,6 +11,7 @@ const resultChannel string = "Result"
 
 // namespace returns the namespace of the service.
 func (service *Service) namespace() []string {
+	return []string{service.Alias}
```

Comment:
has two returns, let's not forget updating

Reply:
alias is not good anyway with the system that we have, it will generate a id too long for docker

Reply:
I revert to use ID here.